.github/actions/get-changelog/dist/.github/actions/get-changelog/index.js

```diff
@@ -1,10 +0,0 @@
-"use strict";
```

Comment:
I guess the changes under the actions should not be touched.

Reply:
good that you spotted this. accidental delete - I tried to revert, but I got two formatting changes I can get rid off.

Reply:
git says the files are the same.